index.html

```diff
@@ -3,14 +3,76 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+
   <title>Great Idea - Responsive I</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+ 
+
 
 </head>
 
-<body>
+
+
+
 <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+<body>
+    <div class="container">
+    <nav>
+        <a class="menu" href="#"> Services</a> 
+        <a class ="menu" href="#">  Product</a>
+        <a class ="menu" href="#">  Vision</a>
+        <a  class ="menu" href="#"> Features</a>
+        <a class ="menu" href="#">  About</a>
+        <a class ="menu" href="#"> Contact</a>
+      </nav>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+          <h1> Innovation </br>
+          On </br>
+          Demand </h1>
+      
+      <h2 class="start"> Get Started</h2>
+      
+        <img class="pic" src="img/header-img.png" alt="Image of a code snippet.">
+  
+    <h3 class="funkychicken">Features</h3>
```

Comment:
the class names are funny. I'm fine with them right now as long as you make sure to be aware of your environment and team. And honestly, if you come back to your code later, you won't have to work through what your class names meant and take that time catching up.